app/build.gradle

```diff
@@ -41,6 +41,7 @@ android {
             minifyEnabled true
             shrinkResources true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
```

Comment:
Remove this one.

Reply:
WHOOPS sorry. oversight.
also note I resolved conflicts.